configure-oidc.md

```diff
@@ -75,3 +75,128 @@ When issuing a `cosign sign` command, add the flags to pass in the client secret
+
+#### Option 1: Create a KeycloakRealm in OpenShift on the CLI
+
+Create a Custom Resource file from which to create the KeycloakRealm that includes the Github identity provider:
+
+```shell
+$ cat realm-cr.yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakRealm
+metadata:
+  labels:
+    app: sso
+  name: sigstore
+spec:
+  instanceSelector:
+    matchLabels:
+      app: sso
+  realm:
+    displayName: Sigstore
+    enabled: true
+    id: sigstore
+    realm: sigstore
+    sslRequired: none
+    identityProviders:
+      - alias: github
+        displayName: Github
+        enabled: true
+        firstBrokerLoginFlowAlias: first broker login
+        internalId: github
+        providerId: github
+        trustEmail: true
+        config:
+            clientId: <my client id> # add your client ID
+            clientSecret: <my client secret> # add your client secret
+```
+
+Make sure that you have no existing KeycloakRealm with the `metadata.name` specified in your `realm-cr.yaml` file. IMPORTANT: make sure you are okay with deleting your current `sigstore` KeycloakRealm.
+```shell
+$ oc delete keycloakrealm sigstore -n keycloak-system
+keycloakrealm.keycloak.org "sigstore" deleted
+```
+
+Create the new KeycloakRealm that includes the Github identity provider from the custom resource file:
+```shell
+$ oc create -f realm-cr.yaml -n keycloak-system
+keycloakrealm.keycloak.org/sigstore created
+```
+
+#### Option 2: 
+You may also add the Github Identity provider to an existing KeycloakRealm through the Keycloak console by selecting `Identity Providers` in the lefthand menu bar and selecting `Add provider -> Github` from the drop down menu. See above for instructions on how to access the Keycloak console.
+
+### Copy the Redirect URI to your Github OAuth App
+
+From the Keycloak console, navigate to `Identity Providers` in the lefthand menu bar, and select the provider you have just created: `github`. It should look like this:
+
+<p align="center">
+  <img src="/images/github_idp.png" />
+</p>
+
+From this page, copy the Redirect URI:
+
+<p align="center">
+  <img src="/images/redirect_uri.png" />
+</p>
+
+Paste the redirect URI value into the OAuth app Authorization Callback URL field, then click `Update Application`:
+<p align="center">
+  <img src="/images/github_callback.png" />
+</p>
+
+### Add IdentityProviderMapper
+
+From the Keycloak console, navigate to `Identity Providers` in the lefthand menu bar, and select the provider you have just created: `github`. Select the tab `Mappers` and on the right-hand side, click the `create` button.
+
+Set the field values as shown below. The 'name' can be anything, as this will be overwritten.
+
+<p align="center">
+  <img src="/images/mapper2.png" />
+</p>
+
+Make sure to hit `save`.
+
+### Sign the artifact
+
+Sign the artifact using the same commands listed in [`sign-verify.md`](https://github.com/securesign/sigstore-ocp/blob/release-1.0.beta/sign-verify.md):
```

Comment:
Recommend using relative pathing which will always align to the branch that the file is currently on